dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/utils/ProcessUtils.java

```diff
@@ -283,10 +275,8 @@ private static boolean needsEscaping(int verificationType, String arg) {
    * @param logger      logger
    * @param tenantCode  tenant code
    * @param executePath     execute path
-   * @throws IOException io exception
    */
-  public static void cancelApplication(List<String> appIds, Logger logger, String tenantCode,String executePath)
-          throws IOException {
+  public static void cancelApplication(List<String> appIds, Logger logger, String tenantCode,String executePath) {
```

Comment:
```suggestion
  public static void cancelApplication(List<String> appIds, Logger logger, String tenantCode, String executePath) {
```
Hi,
Please configure the checkstyle.xml and codestyle plugin to your ide. It will help to format the code.
[1] https://github.com/apache/incubator-dolphinscheduler/blob/dev/style/checkstyle.xml
[2] https://github.com/apache/incubator-dolphinscheduler/blob/dev/style/intellij-java-code-style.xml

Reply:
OK，I'll do it. I'll re submit the code